blocks/video-player-block/features/video-player/default.jsx

```diff
@@ -128,7 +128,7 @@ function VideoPlayer({ customFields = {}, embedMarkup }) {
 				embedMarkup: formatPowaVideoEmbed(powaMarkup, {
 					autoplay,
 					playthrough,
-					"aspect-ratio": 1 / aspectRatio, // format aspect is reversed
+					"aspect-ratio": "0.562", // format aspect is reversed
```

Comment:
The Video Player block below this should work with the numeric aspects since it uses the Video component that just applies it to the CSS `aspect-ratio` variable property.  Was this causing an issue?

https://github.com/WPMedia/arc-themes-components/blob/arc-themes-release-version-2.0.3/src/components/video/index.jsx#L34

https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#examples_of_values_for_aspect-ratio

Which supports numeric or "css property delimited" values (1, 0.562, or "16 / 9")

Reply:
https://w3c.github.io/csswg-drafts/css-values-4/#ratios

Reply:
@nschubach it was causing failures in chromatic, since the updated video component transforms the aspect ratio prop by splitting it at the ":". I can update the video component to handle different aspect ratios if you'd like